paperspace/commands/experiments.py

```diff
@@ -168,3 +175,77 @@ def get_experiment_details(experiment_id, api=experiments_api):
             logger.error("Error parsing response data")
 
     log_response(response, details, "Unknown error while retrieving details of the experiment")
+
+
+class ExperimentLogsCommand(common.CommandBase):
+    last_line_number = 0
+    base_url = "/jobs/logs"
+
+    is_logs_complete = False
+
+    def execute(self, experiment_id, line, limit, follow):
+        if follow:
+            self.logger.log("Awaiting logs...")
+
+        self.last_line_number = line
+        table_title = "Experiment %s logs" % experiment_id
+        table_data = [("JOB ID", "LINE", "MESSAGE")]
+        table = terminaltables.AsciiTable(table_data, title=table_title)
+
+        while not self.is_logs_complete:
+            response = self._get_logs(experiment_id, self.last_line_number, limit)
+
+            try:
+                data = response.json()
+                if not response.ok:
+                    self.logger.log_error_response(data)
+                    return
+            except (ValueError, KeyError) as e:
+                if response.status_code == 204:
+                    continue
+                self.logger.log("Error while parsing response data: {}".format(e))
+                return
+            else:
+                self._log_logs_list(data, table, table_data, follow)
+
+            if not follow:
+                self.is_logs_complete = True
+
+    def _get_logs(self, experiment_id, line, limit):
+        params = {
+            'experimentId': experiment_id,
+            'line': line,
+            'limit': limit
+        };
+        return self.api.get(self.base_url, params=params)
+
+    def _log_logs_list(self, data, table, table_data, follow):
+        if not data:
+            self.logger.log("No Logs found")
```

Comment:
if we `return` right after that, following `else` wouldn't be needed and it would spare us one indentation
#stylisticnazi